ext/scripts/list_newest_versions/r_latest_versions.r

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env Rscript
+library(dplyr)
+library(RWsearch)
+
+args = commandArgs(trailingOnly=TRUE)
+if (length(args)!=2) {
+  stop("Usage: ./r_latest_versions.r <input_package_list_file> <output_package_list_file>  .n", call.=FALSE)
+}
+input_file = args[1]
+output_file = args[2]
+
+input = read.table(file = input_file, sep = "|", comment.char = "#")
+packages = input[,1]
+
+#archivedb_down(filename = "CRAN-archive.html", dir = tempdir(), url = "https://cran.r-project.org/src/contrib/Archive")
+#packages=c("RCurl","tidyverse")
+#archive_version = p_archive_lst(packages)
+#previous_versions = archive_version[["RCurl"]][["Name"]]
```

Comment:
are the comment here left for future reference?

Reply:
Future reference

Reply:
Created a ticket for that https://github.com/exasol/script-languages-release/issues/469